code/modules/mob/living/carbon/xenomorph/castes/widow/widow.dm

```diff
@@ -28,3 +28,19 @@
 	M.layer = initial(M.layer)
 	M.pixel_x = rand(-8, 8)
 	M.pixel_y = rand(-8, 8)
+
+/mob/living/carbon/xenomorph/widow/transfer_to_hive(hivenumber)
+	. = ..()
+	var/mob/living/carbon/xenomorph/widow/X = src
+	var/datum/action/xeno_action/create_spiderling/create_spiderling_action = X.actions_by_path[/datum/action/xeno_action/create_spiderling]
+	for(var/mob/living/carbon/xenomorph/spider in create_spiderling_action.spiderlings)
+		spider.transfer_to_hive(hivenumber)
+
+/mob/living/carbon/xenomorph/widow/on_eord(turf/destination)
+	..()
+	var/mob/living/carbon/xenomorph/widow/X = src
+	var/datum/action/xeno_action/create_spiderling/create_spiderling_action = X.actions_by_path[/datum/action/xeno_action/create_spiderling]
+	for(var/mob/living/carbon/xenomorph/spider in create_spiderling_action.spiderlings)
+		spider.revive(TRUE)
+		INVOKE_ASYNC(spider, TYPE_PROC_REF(/atom/movable, forceMove), destination)
```

Comment:
Why async

Reply:
Because that's what the original teleport proc uses for EORG, so I used it the same way just in case

Reply:
Can you like uhh not?

Reply:
Made it simply use forceMove